pkg/sessions/redis/redis_store_test.go

```diff
@@ -12,11 +15,49 @@ import (
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/options"
 	sessionsapi "github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/logger"
+	redissession "github.com/oauth2-proxy/oauth2-proxy/pkg/sessions/redis"
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/sessions/tests"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/assert"
 )
 
+func TestLegacyV5DecodeSession(t *testing.T) {
```

Comment:
Since the rest of this test suite uses Ginkgo and Gomega, have you considered using either of those? 

You can use gomega assertions withiout Ginkgo by initialising a new Gomega

```
g := NewWithT(t)
err := ...
g.Expect(err).ToNot(HaveOccurred())

t.Run("", func (t *testing.T) {
  // Use a subtest gomega to fail just the subtest
  gs := NewWithT(t)
  ...
})
```

Not a blocker, but I would prefer the consistency